lessons/Projects/project-snack-habits/langs/en-us/index.adoc

```diff
@@ -0,0 +1,194 @@
+- When your class is done collecting data, 
+  * duplicate the sheet in a new tab 
+  * name the new tab "cleaned" 
+  * remove the timestamp column and the image column
+  * make sure the sharing settings are set to "Anyone with the link can view"
+@ifproglang{pyret}{
+  * copy the url into the @starter-file{snack} to import the data, publish the file and share a link with your students
+}
+@ifproglang{codap}{
+  * import the spreadsheet into CODAP and share a link with your students
+}
+
+We have set the form up with data verification for most questions to minimize the amount of cleaning you will have to do, but we would expect that the one word answers to "Why are you eating this snack?" will require some attention to address: 
+
+- typos
+- inconsistent capitalization
+- words that mean the same thing as each other and should be combined 
+- perhaps there are a number of nonsense words that might make more sense to be replaced by "idk" why I'm snacking?
+
+If you have the time, we encourage you to project the spreadsheet and tackle cleaning the data with your students, asking:
+
+- What inconsistencies do you see in the data?
+- How should we address them?
+
+Ideally, by the time you're done you could make a pie-chart of the "why" column and the breakdown of reasons would be informative. 
+}
+
+@lesson-instruction{
+- Now that we have finished collecting our data, let's see what we can learn about our habits!
+- Open the starter file I shared with you and use it to complete @handout{data-cycle-categorical.adoc}
+}
+
+@teacher{@right{@image{images/pie-chart-salty-sweet.png, 500}} For reference, here's an example of a display from one Bootstrap teacher's class. Consider the kinds of discussions your class might have if the data from your class came back looking like this.}
+
+=== Synthesize
+
+- What did you learn?
+- What new questions do you have?
+
+== Research: U.S. Snacking Habits Data
+
+=== Overview
+
+Students will gather information from studies about US American snacking habits and compare them to the data we've just gathered as a class.
+
+=== Launch
+
+@lesson-instruction{
+Do you think the sample we've made of our snack habits is representative of snacking in the United States?
+}
+
+We don't have to guess! There's plenty of research out there for us to look at. In fact, enough data has been collected about childhood obesity in the United States that soda machines and unhealthy snacks have been pulled from many schools.
+
+=== Investigate
+
+@lesson-instruction{
+- Open your favorite search engine (Google, Brave, DuckDuckGo, etc.) and type +
+"What do Americans eat for snack?”
+- Look for graphs and statistics that relate to the kinds of data we're collecting about our own snacking habits.
+}
+
+=== Synthesize
+
+Compare and contrast your findings from our class data and the research you've been doing about U.S. Data. 
+
+@scrub{Make a similar and different handout}
+
```

Comment:
Was just going to add a comment that this requires a handout then I saw the `@scrub`. 

Anyhow... I think a handout is needed. I would want my kids to fill it out in groups, to discuss it, to maybe even include some questions to help guide their thinking.

See my notes in the next section re: why this is useful.